modules/vaos/spec/request/v2/appointments_request_spec.rb

```diff
@@ -722,24 +722,25 @@ def stub_clinics
         end
 
         it 'updates the service name, physical location, friendly name, and location' do
-          appointment = { clinic: 'Test Clinic', location_id: 1 }
-
+          stub_facilities
           allow_any_instance_of(VAOS::V2::MobileFacilityService).to receive(:get_clinic)
             .and_return(service_name: 'Service Name', physical_location: 'Physical Location')
-          allow_any_instance_of(VAOS::V2::MobileFacilityService).to receive(:get_facility).and_return('Location')
-          allow_any_instance_of(described_class).to receive(:appointment).and_return(appointment)
-          allow(Rails.logger).to receive(:info).at_least(:once)
+          VCR.use_cassette('vaos/v2/appointments/get_appointment_200',
```

Comment:
Needed to update the test to use more realistic traffic and mock facility due to logging and post processing expectations.